src/cli/add.rs

```diff
@@ -58,6 +58,8 @@ use crate::{
 ///   array
 /// - `pixi add --pypi boto3 --feature aws` will add `boto3` to the
 ///   `dependency-groups.aws` array
+/// - `pixi add --pypi --editable 'boto3 @ file://path/to/boto3'` will add
+///   the local editable `boto3` to the `project.dependencies` array
```

Comment:
```suggestion
///   the local editable `boto3` to the `pypi-dependencies` array
```

Reply:
because of this it might not really be the right location for this example. As adding `--editable` will move it into the pypi-dependencies